src/frontend/hooks/goose-form.js

```diff
@@ -81,13 +81,25 @@ const useGooseForm = () => {
 
   const saveForm = useCallback(async () => {
     // TODO: Validation checks here.
-
     // Send data to server:
+
     try {
+      // files added 
       const files = images;
```

Comment:
Can we name this variable to something different (e.g., `newlyAddedImages`)? The current variable name of `files` isn't very descriptive